odoo/tests/common.py

```diff
@@ -702,15 +702,24 @@ def executable(self):
                     return bin_
 
         elif system == 'Windows':
-            # TODO: handle windows platform: https://stackoverflow.com/a/40674915
-            pass
+            bins = [
+                '%ProgramFiles%\Google\Chrome\Application\chrome.exe',
+                '%ProgramFiles(x86)%\Google\Chrome\Application\chrome.exe',
+                '%LocalAppData%\Google\Chrome\Application\chrome.exe',
+            ]
+            for bin_ in bins:
+                bin_ = os.path.expandvars(bin_)
+                if os.path.exists(bin_):
+                    return bin_
 
         raise unittest.SkipTest("Chrome executable not found")
 
     def _spawn_chrome(self, cmd):
-        if os.name != 'posix':
-            return
-        pid = os.fork()
+        if os.name == 'nt':
+            proc = subprocess.Popen(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
In the posix branch we don't actually redirect stdout, only stderr.

Reply:
Ok, fixed